flocker/control/test/test_httpapi.py

```diff
@@ -1960,13 +1963,54 @@ def saved(ignored):
         'primary address, but the input schema currently requires the primary '
         'attribute. This will need to be addressed before or as part of '
         'https://clusterhq.atlassian.net/browse/FLOC-1404'
+        # Consider fixing this issue before implementing the resize API changes.
```

Comment:
Probably makes as much sense to do it in this branch as in a FLOC-1404 branch (FLOC-1404 just being the metadata update issue).